Resources/Prototypes/_Sunrise/Entities/Objects/Fun/figurines.yml

```
@@ -21,3 +21,51 @@
+    sprite: _Sunrise/Objects/Fun/figurines.rsi
+    state: cburn
+  - type: SpeakOnTrigger
+    pack: FigurinesCburn
+
+- type: entity
+  parent: BaseFigurine
+  id: ToyFigurineDeathsquad
+  name: Deathsquad figure
+  description: A figurine depicting.... Who's that?
+  components:
+  - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)